library/ecp.c

```diff
@@ -1845,6 +1848,16 @@ int mbedtls_ecp_muladd( mbedtls_ecp_group *grp, mbedtls_ecp_point *R,
     mbedtls_ecp_point_free( &mP );
 
     return( ret );
+
+#else /* ECP_SHORTWEIERSTRASS */
+    (void) grp;
+    (void) R;
+    (void) m;
+    (void) P;
+    (void) n;
+    (void) Q;
+    return( MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE );
```

Comment:
Because the function existed and I don't want to remove a function in a minor version.

Reply:
I'm don't think compatibility considerations apply here, as they would only matter for a non-default configuration that was simply not possible before (and never was). Prior to that change, with only Mongomery curves, you couldn't build ecp.c, so you didn't have that function either. After that change, with only Montgomery curves, you would start having the rest of the module but no that function, which you didn't have anyway. So I think I would prefer compile-time exclusion here.

Note that removing this function at compile-time also means making ECDSA dependent on at least one ECDSA-capable curve (currently, only Short Weierstrass curves) to be defined, because ECDSA uses muladd().